src/System.Linq.Expressions/src/System/Linq/Expressions/DebugViewWriter.cs

```diff
@@ -303,7 +299,6 @@ private void VisitExpressions<T>(char open, char separator, IList<T> expressions
                 case '(': close = ')'; break;
                 case '{': close = '}'; break;
                 case '[': close = ']'; break;
-                case '<': close = '>'; break;
```

Comment:
Dead code indeed. You can check for `VisitExpressions` calls with '<'; none exist.